vertical-pod-autoscaler/pkg/recommender/input/cluster_feeder_test.go

```diff
@@ -380,7 +420,70 @@ func makeTestSpecClient(podLabels []map[string]string) spec.SpecClient {
 	}
 }
 
-func TestClusterStateFeeder_LoadPods(t *testing.T) {
+func newTestContainerSpec(podID model.PodID, containerName string, milicores int, memory int64) spec.BasicContainerSpec {
+	containerID := model.ContainerID{
+		PodID:         podID,
+		ContainerName: containerName,
+	}
+	requestedResources := model.Resources{
+		model.ResourceCPU:    model.ResourceAmount(milicores),
```

Comment:
Typo nit: 
```suggestion
func newTestContainerSpec(podID model.PodID, containerName string, millicores int, memory int64) spec.BasicContainerSpec {
	containerID := model.ContainerID{
		PodID:         podID,
		ContainerName: containerName,
	}
	requestedResources := model.Resources{
		model.ResourceCPU:    model.ResourceAmount(millicores),
```

Reply:
I assume it's a typo copied from vertical-pod-autoscaler/pkg/recommender/input/spec/spec_client_test_util.go

Reply:
done.